dags/merino_jobs.py

```diff
@@ -72,14 +73,37 @@ def merino_job(
             "--gcs-path",
             "moz-fx-data-prod-external-data/contextual-services/merino-jobs/wikipedia-exports",
             "--gcp-project",
-            "moz-fx-data-shared-prod",
+            "moz-fx-data-shared-nonprod",
         ],
         env_vars={
-            "MERINO_ENV": "production",
+            "MERINO_ENV": "staging",
+        },
+    )
+
+    wikipedia_indexer_build_index_for_staging = merino_job(
+        "wikipedia_indexer_build_index",
```

Comment:
I would recommend changing the name of the tasks so they can easily be differentiated in the airflow UI